reporters/loadReporter.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+var getReporter = require('jscs/lib/cli-config').getReporter;
+
+module.exports = function (reporter) {
+	// we want the function
+	if (typeof reporter === 'function') return reporter;
+
+	// load JSCS built-in or full-path or module reporters
+	if (typeof reporter === 'string' || !reporter) {
+		var rpt = getReporter(reporter).writer;
+		if (rpt) return rpt;
+		try {
+			return require(reporter);
```

Comment:
I wonder if we need this?
This covers `jscs.reporter('reporter-npm-package-name')`, but since `loadReport` accepts a function argument, the user code could require the reporter directly: `jscs.reporter(require('reporter-npm-package-name'))`

@sindresorhus what do you think?